Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -122,12 +124,7 @@ public static AssertionScope Current
             private set => SetCurrentAssertionScope(value);
         }
 
-        /// <summary>
-        /// Forces the formatters that support it to add the necessary line breaks.
-        /// </summary>
-        /// <remarks>
-        /// This is just shorthand for modifying the <see cref="FormattingOptions"/> property.
-        /// </remarks>
+        /// <inheritdoc cref="IAssertionScope.UsingLineBreaks"/>
```

Comment:
I wonder if we don't use `inheritdoc`, would you still get the right Intellisense if you use FA as a binary. Because with Rider, while working with the FA code base, it just shows me the docs from the interface.

Reply:
I think that is a Rider trick then, as `UsingLineBreaks` is not inheriting from the interface itself. It's only references by an explicit implementation of the interface (bottom of the file)

In Visual Studio it doesn't show up without the `inheritdoc`.

Reply:
For methods that inherit from the interface, it can be left out as it will inherit implicitly, but it results in a SA message about public elements that should be documented.
So, other then making the behavior and intent more explicit, it also removes some SA messages.